pkg/infra/tower.go

```diff
@@ -46,9 +50,7 @@ func NewTower(maxPlatFood float64, agentCount,
 
 func (t *Tower) Tick() {
 	//logs
```

Comment:
```suggestion
```

Reply:
The fact that you're calling the function `TowerStateLog()` should hopefully make it pretty clear that it is a log